src/components/atoms/UiIcon/UiIcon.stories.ts

```diff
@@ -0,0 +1,179 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { UiIcon } from '@/../index';
+import { defineAsyncComponent } from 'vue';
+import icons from './icons';
+import './UiIcon.stories.scss';
+
+const meta = {
+  title: 'Atoms/Icon',
+  component: UiIcon,
+  argTypes: {
+    icon: {
+      control: { type: 'select' },
+      options: icons,
+    },
+  },
+} satisfies Meta<typeof UiIcon>;
+export default meta;
+type Story = StoryObj<typeof UiIcon>;
+export const Basic: Story = {
+  render: (args) => ({
+    components: { UiIcon },
+    setup() {
+      return { args };
+    },
+    template: '<UiIcon v-bind="args"/>',
+  }),
+};
+Basic.args = { icon: 'absent' };
```

Comment:
When I set it in meta Storybook it will provide force to stories and I can pass anything to these props.